docs/oev-searchers/in-depth/data-feeds/index.md

```diff
@@ -35,6 +35,10 @@ It consists of:
 - Template ID = A template represents an endpoint of the data provider API, e.g.
   ETH/USD.
 
+::: warning ⚠️ Airnode Mnemonic Rotation
+Data providers rotate their Airnode mnemonics every 6 months as part of Api3's security practices. This results in new Airnode addresses being generated. Current Airnode addresses can be found on the [API3 Market](https://market.api3.org) and in the `@api3/api-integrations` package.
```

Comment:
I don't think searchers should go to `@api3/api-integrations` for this. They can refer the market or read them directly on-chain.
```suggestion
Data providers rotate their Airnode mnemonics every 6 months as part of Api3's security practices. This results in new Airnode addresses being generated. Current Airnode addresses can be found on the [API3 Market](https://market.api3.org) and in the `@api3/api-integrations` repository.
```

I think this message should be put in https://docs.api3.org/oev-searchers/in-depth/data-feeds/#dapp-sources which talks about how to obtain the data feed beacons.